Makefile

```diff
@@ -315,6 +315,39 @@ else
 	docker-compose up -d db auth
 endif
 
+$(TMP_PATH)/client-repo:
+	$(GIT_BIN) clone git@github.com:fabric8-services/fabric8-wit-client.git $(TMP_PATH)/client-repo
+
+.PHONY: update-client
+## Updates the client library for this repository in tmp/client-repo
+update-client: $(TMP_PATH)/client-repo $(GOAGEN_BIN) $(TMP_PATH) $(GIT_BIN)
+	cd $(TMP_PATH)/client-repo \
+	&& $(GIT_BIN) fetch origin \
+	&& $(GIT_BIN) reset --hard origin/master
+	# Generate client code
+	$(GOAGEN_BIN) client -d github.com/fabric8-services/fabric8-wit/design --notool --pkg main -o $(TMP_PATH)
+	# Move client code to client-repo
+	mv $(TMP_PATH)/main/* $(TMP_PATH)/client-repo
+	# Generate commit message just in case we need it later
+	echo "Changes that went into the upstream library:" > $(TMP_PATH)/commit_message.txt
+	echo "" >> $(TMP_PATH)/commit_message.txt
```

Comment:
If this is to invoke newline, you could do the same with `echo -e "blah blah\n"` in the previous line.

Reply:
This indeed was to create a new line ;)